query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/select/OngoingSelection.java

```diff
@@ -612,6 +612,54 @@ default Select cast(Selector selector, DataType targetType) {
     return selector(Selector.cast(selector, targetType));
   }
 
+  /**
+   * Shortcut to select the result of the built-in {@code toDate} function.
+   *
+   * @see Selector#toDate(CqlIdentifier)
+   */
+  default Select toDate(CqlIdentifier columnId) {
+    return selector(Selector.toDate(columnId));
+  }
```

Comment:
Only adding shortcuts where they make sense, for example `SELECT now()` is highly unlikely to be useful.